tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java

```diff
@@ -1399,24 +1470,46 @@ private long[] setBestNextAndSameVoteValues(Set<T> sortedValues, HashMap<T, Long
 
     /**
      * Compute the status for the winning value.
+     * See: https://cldr.unicode.org/index/process
      *
-     * @param weight1 the weight (vote count) for the best value
-     * @param weight2 the weight (vote count) for the next-best value
+     * @param O the weight (vote count) for the best value
+     * @param N the weight (vote count) for the next-best value
```

Comment:
per the link (which is a great addition),

   The first value is the optimal value (O).
   The second value (if any) is the next best value (N).

maybe worth including that in the @param -- I couldn't remember what O and N were abbreviations for

Reply:
I'm not sure what O and N are abbreviations for, possibly Optimal and Nextbest?  but they correspond to the algorithm in the link.